app/js/views/service.jsx

```diff
@@ -33,13 +34,16 @@ export default class Service extends BaseView {
       return null;
     }
 
-    switch(this.state.service.type) {
+    switch (this.state.service.type) {
       case 'ip-camera@link.mozilla.org':
         return (<CameraServiceView service={this.state.service}
-                                   foxbox={this.foxbox} />);
+                                   foxbox={this.foxbox}/>);
+      case 'philips_hue@link.mozilla.org':
+        return (<LightServiceView service={this.state.service}
+                                  foxbox={this.foxbox}/>);
       default:
         return (<DefaultServiceView service={this.state.service}
-                                    foxbox={this.foxbox} />);
+                                    foxbox={this.foxbox}/>);
```

Comment:
I think I must have configured my IDE in the past to remove the whitespace before `/>`.
It doesn't change anything once the code is compiled from JSX to plain JS though.